File: data stracture/linear_search.py
arr = []
arr.append(input("Enter name of 1st element: "))
arr.append(input("Enter name of 2nd element: "))
arr.append(input("Enter name of 3rd element: "))
print(arr)

key = input("Enter the searching element of the array: ")
idx = 0
for el in arr:
    if el == key:
        print("ELEMENT IS FOUND at index", idx)
        break
    idx += 1
else:
    print("ELEMENT NOT FOUND")

File: data stracture/linear_search.c
#include<stdio.h>
int main () {
    int n , key , i ;
    printf("enter the number of array ");
    scanf("%d",&n);
    int arr[n];
    printf("enter the elements  of the array");
    for(i=0;i<n;i++){
        scanf("%d",&arr[i]);
    }
    printf("enter the key value ");
    scanf("%d",&key);
    int found=0;
    for(i=0;i<n;i++){
        if (arr[i]==key){
            printf("number %d is find at index %d",&key,&i);
            found = 1;
            break ;
        }
    }
    if(found != 1){
        printf("not founded");
    }
    return 0 ;
}
